src/WebJobs.Script.WebHost/App_Start/AutofacBootstrap.cs

```diff
@@ -2,17 +2,20 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using Autofac;
+using Microsoft.Azure.WebJobs.Script.Config;
 using Microsoft.Azure.WebJobs.Script.WebHost.WebHooks;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost
 {
     public static class AutofacBootstrap
     {
-        internal static void Initialize(ContainerBuilder builder, WebHostSettings settings)
+        internal static void Initialize(ScriptSettingsManager settingsManager, ContainerBuilder builder, WebHostSettings settings)
```

Comment:
We don't want an instance of the settings manager passed in here. With the current logic, the we're registering one way with DI and passing a provided instance to the `WebHostResolver`.

Ideally this would be creating the instance (same one it registers) and passing the same to the `WebHostResolver`. 
Worse case, we want to at least register the instance it receives.